### PR TITLE
feat: Implement update article service layer - EXO-70382 - Meeds-io/MIPs#119

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java [hunk 1]
```diff
@@ -394,7 +390,20 @@ public void publishNews(News newsToPublish, String publisher) throws Exception {
...
+    newsTargetingService.deleteNewsTargets(news, publisher);
 
+    NewsPageObject newsPageObject = new NewsPageObject(NEWS_METADATA_PAGE_OBJECT_TYPE, news.getId(), null);
+    MetadataItem newsMetadataItem = metadataService.getMetadataItemsByMetadataAndObject(NEWS_METADATA_KEY, newsPageObject).stream().findFirst().orElse(null);
+
+    if (newsMetadataItem != null) {
+      Map<String, String> properties = newsMetadataItem.getProperties();
+      if (properties != null) {
+        properties.put(PUBLISHED, String.valueOf(false));
+        properties.remove(NEWS_PUBLISH_DATE);
```
Comment (azayati) [inline content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java:406]: - can we check if we need this property publishDate ?
-> we try to keep only properties needed

File: content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java [hunk 2]
```diff
@@ -1487,6 +1586,11 @@ private News buildArticle(String newsId) throws WikiException {
         if (properties.containsKey(NEWS_VIEWS) && StringUtils.isNotEmpty(properties.get(NEWS_VIEWS))) {
           news.setViewsCount(Long.parseLong(properties.get(NEWS_VIEWS)));
         }
+        if (properties.containsKey(NEWS_ACTIVITY_POSTED)) {
+          news.setActivityPosted(Boolean.valueOf(properties.get(NEWS_ACTIVITY_POSTED)));
```
Comment (azayati) [inline content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java:1607]: I think that this property is stored inverted, now if I check the "post in activity stream" option when creating the article, the related activity is hidden, and vice versa

**Review by azayati — CHANGES_REQUESTED** [review]

File: content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java (hunk 1 above)
Comment (sofyenne) [reply content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java:406]: This property needs to be displayed in the news list view item !

File: content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java (hunk 2 above)
Comment (sofyenne) [reply content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java:1607]: fixed!

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**